docker/base/os/debian/Dockerfile.bullseye

```diff
@@ -0,0 +1,20 @@
+# This is the dockerfile for generating base images for Bullseye
+
+FROM debian:bullseye
```

Comment:
If you want to keep up with the latest, just remove the version number.  Whenever the image is built, it will use `debian:latest`.

Reply:
yes but best practices say to keep at least the major version fixed; anyway no problem removing it if useful ...